_std/__build.dm

```diff
@@ -47,6 +47,7 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define BONUS_POINTS // gives a bunch of starting points to various abilities/uplinks/weapon vendors
 //#define SHUT_UP_AND_GIVE_ME_MEDAL_STUFF // causes has_medal to always return true - good for testing medal rewards etc.
 //#define SHOW_ME_STATUSES // incredibly hacky visible status effects
+#define ME_AND_MY_40_ALT_ACCOUNTS // Override game mode minimum player requirements for testing revs, nukies etc.
```

Comment:
This should be commented out by default, right?

Reply:
GUESS WHO WAS TESTING IT AND FORGOT